cmd/cluster-display/main.go

```diff
@@ -164,19 +164,35 @@ func main() {
 		logrus.WithError(err).Fatal("failed to set up scheme")
 	}
 
-	kubeconfigChangedCallBack := func(e fsnotify.Event) {
-		logrus.WithField("event", e.String()).Info("Kubeconfig changed, exiting to get restarted by Kubelet and pick up the changes")
+	kubeconfigChangedCallBack := func() {
+		logrus.Info("Kubeconfig changed, exiting to get restarted by Kubelet and pick up the changes")
 		interrupts.Terminate()
 	}
 
-	kubeConfigs, err := util.LoadKubeConfigs(o.hiveKubeconfigPath, "", kubeconfigChangedCallBack)
-	if err != nil {
-		logrus.WithError(err).WithField("o.hiveKubeconfigPath", o.hiveKubeconfigPath).Fatal("could not load Hive kube config")
-	}
-	hiveConfig, ok := kubeConfigs[string(api.HiveCluster)]
-	if len(kubeConfigs) != 1 || !ok {
-		logrus.WithError(err).WithField("o.hiveKubeconfigPath", o.hiveKubeconfigPath).Fatalf("found %d contexts in Hive kube config and it must be %s", len(kubeConfigs), string(api.HiveCluster))
+	var hiveConfig *rest.Config
+	if o.hiveKubeconfigPath == "" {
```

Comment:
Tried to "use --hive-kubeconfig to set a flag value for --kubeconfig to support both" but
there is no setter for `o.kubernetesOptions.kubeconfig`. I have to play with the unparsed `args` to make this work, right?

I feel that the current solution might be easier.